lms/djangoapps/courseware/module_render.py

```diff
@@ -377,6 +458,13 @@ def handle_grade_event(block, event_type, event):
 
         dog_stats_api.increment("lms.courseware.question_answered", tags=tags)
 
+        # Fulfill any milestones for this content module now that all the data has been saved
+        _fulfill_content_milestones(
```

Comment:
Also, I know we are checking feature flags inside of _fulfill_content_milestones. If we add another FEATURE flag check here, it might help code readers know that it really only does anything with that feature flag on.